bigframes/session/_io/bigquery/read_gbq_table.py

```diff
@@ -42,17 +42,53 @@
     import bigframes.session
 
 
+def get_information_schema_metadata(
+    bqclient: bigquery.Client,
+    table_id: str,
+    default_project: Optional[str],
+) -> bigquery.Table:
+    job_config = bigquery.QueryJobConfig(dry_run=True)
+    job = bqclient.query(
+        # TODO: better escaping?
+        f"SELECT * FROM `{table_id}`",
+        job_config=job_config,
+    )
+    parts = table_id.split(".")
+    if len(parts) < 3:
+        project = default_project
+        dataset = parts[0]
```

Comment:
TODO: INFORMATION_SCHEMA should always be considered the "dataset" IMO. That way the region stuff gets escaped along with the project ID.